### ISSUE TITLE
Decorative images should have null alt text values

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3181]
## Description  
Decorative images should have null alt text values. If the meaning of an image is conveyed by nearby text then populating the its alt attribute with text that also explains the meaning of the image will result in screen reader users hearing duplicate content which can be confusing. 

For example, images under the "Form Architecture" heading have their meaning explained by the following content which follows these images in the  DOM order. 

"Do
Put forms in one column. Multiple columns disrupt users’ vertical rhythm and completion." 

Don't
Create multiple columns of input fields per page, except for small, related inputs like name “first” and “last.”

As seen below, the current images contain alt text that simply restates the content the images are decorating.

`<img alt="use a single column for forms layout" src="assets/images/documentation/forms/architecture-do.svg">`

`<img alt="don't use multiple columns for forms" src="assets/images/documentation/forms/architecture-dont.svg">`

Other examples can be found on https://clarity.design/documentation/buttons. The Z and F Pattern images use alt text values that are the same as the visible text that is visually and structurally associated with these images. See the example HTML snippet below.

```
<img alt="Z pattern" src="assets/images/documentation/buttons/z_pattern.svg"></div></div><div class="clrweb-DoxMedia-text">
<h6>Z Pattern</h6>
```

## Location
https://clarity.design/documentation/forms and site wide

## Priority     
Low

## Recommended fix      
Use null alt attribute values for all decorative images as seen below.

`<img alt="" src="assets/images/documentation/forms/architecture-do.svg">`

`<img alt="" src="assets/images/documentation/forms/architecture-dont.svg">`

## LOE  
Low
